sw/otbn/crypto/p384_isoncurve.s

```diff
@@ -145,14 +131,10 @@ p384_isoncurve:
  * This routine raises a software error and halts operation if the curve point
  * is invalid.
  *
- * @param[in]  dmem[12]: dptr_rhs, pointer to dmem location where right hand
- *                               side result rhs will be stored
- * @param[in]  dmem[16]: dptr_lhs, pointer to dmem location where left hand
- *                               side result lhs will be stored
- * @param[in]  dmem[20]: dptr_x, pointer to dmem location containing affine
- *                               x-coordinate of input point
- * @param[in]  dmem[24]: dptr_y, pointer to dmem location containing affine
- *                               y-coordinate of input point
+ * @param[in]  x20: dptr_x, pointer to dmem location containing affine
+ *                          x-coordinate of input point
+ * @param[in]  x21: dptr_y, pointer to dmem location containing affine
+ *                          y-coordinate of input point
```

Comment:
Same note here about the pointers; I'd put the `la` instructions inside the subroutine and use `dmem[x]` and `dmem[y]` directly here.